functions/http/1.3/index.js

```diff
@@ -0,0 +1,67 @@
+import Liquid from './liquid.min';
+
+const engine = new Liquid();
+
+const parseHeaders = (headers) =>
+  headers.reduce((acc, { key, value }) => ({ ...acc, [key]: value }), {});
+
+const parseQueryParameters = (queryParameters) =>
+  queryParameters
+    .map(({ key, value }, index) => {
+      const paramKey = index === 0 ? `?${key}` : key;
+      return `${paramKey}=${encodeURIComponent(value)}`;
+    })
+    .join('&');
+
+const parseLiquid = (body, bodyParameters) =>
+  engine.parseAndRenderSync(
+    body,
+    bodyParameters.reduce(
+      (parameter, { key, value }) => ({ ...parameter, [key]: value }),
+      {},
+    ),
+  );
+
+const generateUrl = (url, protocol, queryParameters) =>
+  `${protocol}://${url}${parseQueryParameters(queryParameters)}`;
+
+const isJson = (data) => {
```

Comment:
maybe let this function just return the parsed json (because not you are parsing it twice)

Reply:
I didn't change anything to the `index.js` file, just added two new types to the output option in the `functions.json`. I'm not going to refactor existing code in this pull request, so I would suggest to create another one if that's desired.